python/orca/src/bigdl/orca/data/shard.py

```diff
@@ -633,6 +644,19 @@ def zip(self, other):
                               "The two SparkXShards should have the same number of elements "
                               "in each partition")
 
+    def fillna(self, value: Union[int, float, str, bool],
+               columns: Optional[Union[str, List[str]]]):
+        if self._get_class_name() == 'pandas.core.frame.DataFrame':
+            import pandas as pd
+            df = self.to_spark_df()
```

Comment:
why do we need to convert to spark df? I think we can simply do a `mapPartition` and call Pandas `fillna`, yes?

Reply:
yes, the original design of this method wanted to leverage Friesian fillna with performance. but it is not necessary.  closed this PR